app/views/documents/_filter_selections.mustache

```diff
@@ -1,48 +1,48 @@
 <span class="count">{{result_count}}</span> <strong>{{pluralized_result_type}}</strong>
```

Comment:
I'm pretty sure this is another admin only piece of code that should be reverted, based on previous PRs against it, eg https://github.com/alphagov/whitehall/pull/4334

Reply:
Pagination for the admin area, indeed. Have reverted this.